Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM harbor.ssk8s.vibrenthealth.com/dockerhub/openshift/origin-release:golang-1.13 AS build-env
+ARG FIRST_FROM_IMAGE=harbor.ssk8s.vibrenthealth.com/dockerhub/openshift/origin-release:golang-1.13
+ARG SECOND_FROM_IMAGE=harbor.ssk8s.vibrenthealth.com/dockerhub/coolbeevip/ubi8-ubi-minimal:latest
```

Comment:
A thought, but would it be slightly more betterer to use a single variable `DOCKER_REGISTRY` and then change the code to do something like
```
ARG OS_DOCKER_REGISTRY=harbor.ssk8s.vibrenthealth.com/dockerhub/openshift
ARG RH_DOCKER_REGISTRY=harbor.ssk8s.vibrenthealth.com/dockerhub/coolbeevip
FROM ${OS_DOCKER_REGISTRY}/origin-release:golang-1.13
```
then switch the jenkins code to `--build-arg DOCKER_REGISTRY=registry.ci.openshift.org/` ?
It's an idea, but I guess the only benefit is that it allows you to keep the registry separate from the actual image name. Since it's controlled in the same repository, it probably does not matter.

It also would have been simpler if the `coolbeevip` registry name stayed with the same pattern.

Reply:
They both have different image names so this would not work.